Package.swift

```diff
@@ -273,7 +273,7 @@ let package = Package(
             dependencies: ["swift-build", "swift-package", "swift-test", "swift-run", "Commands", "Workspace", "SPMTestSupport"]),
         .testTarget(
             name: "WorkspaceTests",
-            dependencies: ["Workspace", "SPMTestSupport"]),
+            dependencies: ["Workspace", "SPMTestSupport", "Commands"]),
```

Comment:
this likely points to some leaky abstraction - we should not need to bring Commands into workspace tests